config/betterquesting/DefaultQuests.json

```diff
@@ -12884,7 +12719,7 @@
           "ismain:1": 1,
           "repeat_relative:1": 1,
           "name:8": "The Pyrolyse Oven",
-          "desc:8": "This machine can produce various Coal products on top of doubling up as a Coke Oven. Both are good for fuel. Additionally, Wood Vinegar is very useful for various chemicals, and Fermented Biomass is a good source of Ethanol.\n\nMaking Wood Tar, and distilling it into Benzene for use in Gas Turbines is also very advisable.\n\nMore coils needed? Yes, batch craft a lot of Mica Insulation for future Coils as well. The Pyrolyse Oven accepts better coils to speed up the recipes."
+          "desc:8": "This multiblock can produce various Coal products on top of doubling up as a Coke Oven. Unless you plan to entirely rely on oil, the Pyrolyse Oven is essential.\nDepending on the configuration, it can output varied fluids/gases.\n\nThe Pyrolyse Oven needs coils, just like your beloved EBF. Batch craft a lot of Mica Insulation Foils for future coils. The Pyrolyse Oven will also accept better coils to increase its processing speed. Some recipes will also accept Nitrogen for faster processing speed.\n\n\n\nIt\u0027s probably about time we talk about the difference between the Distillery, and Distillation Tower tabs in JEI.\nEssentially, the single block Distillery acts as a budget Distillation Tower, except you can only select one out of all the fluid outputs.\nDistillation Tower becomes available at EV tier, and is fairly big and expensive. But that\u0027s something to keep in mind for future automation purposes.\n\nAll the byproducts from the Pyrolyse Oven can be distilled. Here\u0027s a list of them:\n\n- Fermented Biomass: Useful for Ethanol\n- Creosote: Not really useful, other than for lubricant and burning directly in the large boiler\n- Coal Tar: Useful starting EV age for rocket fuel\n- Wood Gas: Contains mostly Carbon Dioxide and Carbon Monoxide\n- Wood Vinegar: Decent source of Methanol\n- Wood Tar: Good source of Benzene\n- Charcoal Byproducts: Distills into all three previous fluids, but requires the Distillation Tower to be worth it\n- Heavy Oil: A way to get a little bit of oil from wood, you lose on the charocal"
```

Comment:
Coal Tar descripton updated. Also fixed "charocal" typo.
You see, I am pretty certain Heavy Oil route from Pyrolyse **is** viable as a main oil source for hydrocarbons, and probably even power if you throw the heavy fuel directly into boilers. And that's why I consider it a little bit of a problem. But at the same time I don't want to lie to players saying it's bad because we want it to be bad.

Reply:
I see what you mean. Let's leave it alone for the time being.